### PR TITLE
Setup push on deploy to pypi

### PR DESCRIPTION
**Pull request by Surgo**
- Set up push on deploy via [travis](https://docs.travis-ci.com/user/deployment/pypi/)
- Deploy only tagged push

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -49,5 +49,15 @@ install:
 - pip install -U pip setuptools tox wheel
 script:
 - tox -e "${TOX_ENV}"
+deploy:
+  provider: pypi
+  user: Surgo
```
Comment (Surgo) [inline .travis.yml]: If you want to use your account, please put here.

- [Install and configure travis CLI](https://github.com/travis-ci/travis.rb#installation)
- Get encrypted password from `travis encrypt {your pypi password}`

File: .travis.yml [hunk 2]
```diff
@@ -49,5 +49,15 @@ install:
...
+  password:
+    secure: ECQgP3QvjNyqe7sqeKMHOu4wAAvuDobweIGfHXMuWmLer7FKJeCb0HhuSNzZjXXCdTiE1km7dSaYU1Uvb0aA8Zi29oOS0oR4qPVKFibNSqnWrc8WnPhvSCnK9cT6CtwSpK48KYqENHfKbr1SAR2vTX8J3NhwRqBB0aS6bCYEKWU=
+  distributions: sdist bdist_wheel
+  on:
+    repo: michaeljohnbarr/django-timezone-utils
+    tags: true
```
Comment (Surgo) [inline .travis.yml]: Only tagged push.

File: .travis.yml [hunk 3]
```diff
@@ -49,5 +49,15 @@ install:
...
 script:
 - tox -e "${TOX_ENV}"
+deploy:
+  provider: pypi
+  user: Surgo
+  password:
+    secure: ECQgP3QvjNyqe7sqeKMHOu4wAAvuDobweIGfHXMuWmLer7FKJeCb0HhuSNzZjXXCdTiE1km7dSaYU1Uvb0aA8Zi29oOS0oR4qPVKFibNSqnWrc8WnPhvSCnK9cT6CtwSpK48KYqENHfKbr1SAR2vTX8J3NhwRqBB0aS6bCYEKWU=
+  distributions: sdist bdist_wheel
+  on:
+    repo: michaeljohnbarr/django-timezone-utils
```
Comment (Surgo) [inline .travis.yml]: Only this repo.

File: .travis.yml [hunk 4]
```diff
@@ -49,5 +49,15 @@ install:
...
+    tags: true
+    condition: "$TOX_ENV = py37-dj21"
```
Comment (Surgo) [inline .travis.yml]: Only `py37-dj21` tox env

File: setup.cfg [hunk 5]
```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal = 1
```
Comment (Surgo) [inline setup.cfg]: Support both of py2 and py3

File: .travis.yml (hunk 1 above)
Comment (michaeljohnbarr) [reply .travis.yml]: I'm good with using yours. Looks like it is all set up. 😃

**Comment by michaeljohnbarr** [comment]
Quick question (pardon my ignorance): Merging this to master won't break/try to re-submit our current release, right? That's what the [`tags: true`](https://docs.travis-ci.com/user/deployment/pypi/#deploying-tags) directive does, I believe?

**Comment by Surgo** [comment]
We can test deploy like `0.13-pre` or `0.13-alpha` tag.

**Comment by Surgo** [comment]
Ahhh! It can test with tagging this branch to `0.13-pre`

**Comment by pjdelport** [comment]
Nice! (I'll have to look at setting this up for some of my other projects, too…)

**Comment by michaeljohnbarr** [comment]
@Surgo - I'm happy for you to go ahead and merge/tag for testing. Feel free.

**Comment by Surgo** [comment]
Pushed tag `0.13-pre` for testing

- Tagged [0.13-pre](https://github.com/michaeljohnbarr/django-timezone-utils/releases/tag/0.13-pre)
- [Travis build](https://travis-ci.org/michaeljohnbarr/django-timezone-utils/builds/435442354) for a `0.13-pre` release
- [Deploy log for pypi](https://travis-ci.org/michaeljohnbarr/django-timezone-utils/jobs/435442365#L582-L658)
- Its added source and wheel package to [pypi](https://pypi.org/project/django-timezone-utils/0.13/#files)

    - `django_timezone_utils-0.13-py2.py3-none-any.whl (9.7 kB)`
    - `django-timezone-utils-0.13.tar.gz (7.9 kB)`

I'm sorry but 0.13 wheels had duplicated... I had to change tag version. 

screenshot 2018-10-01 14 30 48

**Comment by Surgo** [comment]
@michaeljohnbarr Its works fine for me.

**Pull request merged**